minio/api.py

```diff
@@ -612,7 +614,7 @@ def fget_object(self, bucket_name, object_name, file_path, request_headers=None)
         # Return the stat
         return stat
 
-    def get_object(self, bucket_name, object_name, request_headers=None):
+    def get_object(self, bucket_name, object_name, request_headers=None, sse=None):
         """
```

Comment:
Here you should check if the see parameter passed is `SSE-C`, if not you should error out I guess